packages/theming/apple-theme/src/appleHighContrast.macos.ts

```diff
@@ -0,0 +1,9 @@
+export let isHighContrastEnabled = false;
+
+export function setIsHighContrast(isHighContrast: boolean) {
+  isHighContrastEnabled = isHighContrast;
+}
+
+export function getIsHighContrast() {
+  return isHighContrastEnabled;
+}
```

Comment:
For my own knowledge, why did you have to separate this out to it's own file? It seems like it was to resolve a circular dependency?

Reply:
Yeah, I had to extract the dependency part so that the caller doesn't query from the same file.